www/src/data/guidelines/logo/footnotes.yaml

```
@@ -1,10 +1,10 @@
 - description: Kyle Mathews considering Futura
   href: "https://github.com/gatsbyjs/gatsby/issues/408#issuecomment-245157645"
-- description: Sacha Greif posting the initial logo design
+- description: Sacha Grief posting the initial logo design
```

```suggestion
- description: Sacha Greif posting the initial logo design
```
that's a person name, should not be touched